@navikt/core/react/src/form/combobox/SelectedOptions/SelectedOptions.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Chips } from "../../..";
+
+interface SelectedOptionsProps {
+  selectedOptions?: string[];
+  handleDeleteSelectedOption: (option: string) => void;
+  children: React.ReactNode;
+}
+
+const SelectedOptions: React.FC<SelectedOptionsProps> = ({
+  selectedOptions = [],
+  handleDeleteSelectedOption,
+  children,
+}) => {
+  return (
+    <Chips className="navds-combobox__selected-options">
+      {selectedOptions.length
+        ? selectedOptions.map((option, i) => (
+            <Chips.Removable
+              className="navds-combobox__selected-option"
+              key={option + i}
+              onMouseUp={(e) => {
```

Comment:
Tror denne kan byttes ut med `onPointerUp` for enda bedre device-støtte (https://javascript.info/pointer-events)

Reply:
Det kunne den. Har gjort det alle steder onMouseXXX ble brukt (scopet til combobox-mappen).